apps/server/src/infra/etherpadClient/etherpad-client.module.ts

```diff
@@ -0,0 +1,29 @@
+import { HttpModule, HttpService } from '@nestjs/axios';
+import { DynamicModule, Global, Module } from '@nestjs/common';
+import { Logger, LoggerModule } from '@src/core/logger';
+import { EtherpadRestClient } from './etherpad-rest-client';
+import { EtherpadRestClientOptions } from './etherpad-rest-client-options';
+
+@Global()
+/**
+ * @Global is used here to make sure that the module is only instantiated once, with the configuration and can be used in every module.
+ * Otherwise, you need to import the module with configuration in every module where you want to use it.
+ */
```

Comment:
Please move this comment above the annotation as it nows feels a little bit disconnected from the other class' annotations.

```suggestion
/**
 * @Global is used here to make sure that the module is only instantiated once, with the configuration and can be used in every module.
 * Otherwise, you need to import the module with configuration in every module where you want to use it.
 */
@Global()
```

Reply:
done